cdk/l3-cdk-resources/atlas-basic/src/index.ts

```diff
@@ -1,98 +1,414 @@
 import * as atlas from '@mongodbatlas-awscdk/cluster';
 import * as user from '@mongodbatlas-awscdk/database-user';
+import * as project from '@mongodbatlas-awscdk/project';
 import * as ipAccessList from '@mongodbatlas-awscdk/project-ip-access-list';
 import * as cdk from 'aws-cdk-lib';
-// import * as cloudformation from 'aws-cdk-lib/aws-cloudformation';
 import { Construct } from 'constructs';
-import * as defaults from './defaults.json';
 
+/** @type {*} */
+const defaults = {
+  projectName: 'cdk-project',
+  clusterName: 'cdk-cluster',
+  dbName: 'admin',
+  username: 'cdk-user',
+  password: 'cdk-pwd',
```

Comment:
Just for my knowledge, at what point would these credentials come into use? Would the user need to know about these at all?

Reply:
Sure. I should add this to Readme.

Reply:
Updated Readme.